MyOnboarding/test/MyOnboardingApp.Api.Tests/TodoListControllerTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using MyOnboardingApp.Api.Controllers;
+using MyOnboardingApp.Api.Models;
+using NUnit.Framework;
+
+namespace MyOnboardingApp.Tests
+{
+    [TestFixture]
+    public class TodoListControllerTest
```

Comment:
You should name the tests using What-When-Then structure => Get_IdSpecified_ReturnsItemWithId. So you can correctly specify the behavior and everyone can see, what you are trying to test.

Reply:
Also, you should rewrite your tests to check correct status code is returned once you fix the controller 😉